types/ApiClient.d.ts

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+type RequestOptions = {
+    headers?: Record<string, string>;
+    body?: Record<string, string>;
```

Comment:
this could be a string or and array too, maybe a separate ticket?. One good exercise would be to check all request options that we have at https://api.mattermost.com

Reply:
Good point! I've created https://mattermost.atlassian.net/browse/MM-30180